### PR TITLE
Display drivers

### PR DESCRIPTION
**Pull request by agriffard**
Some drivers (Part and Workflows) are called `...Display` and some `...DisplayDriver`.

Trying to be explicit (instead of keeping the names short).

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.Contents/Workflows/Drivers/ContentCreatedEventDisplayDriver.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+using OrchardCore.ContentManagement.Metadata;
```
Comment (hishamco) [inline src/OrchardCore.Modules/OrchardCore.Contents/Workflows/Drivers/ContentCreatedEventDisplayDriver.cs:1]: Is the aim of this PR to rename or add new display driver(s)  too?

**Review by hishamco — COMMENTED** [review]
LGTM but this is sort of breaking changes, but it's acceptable while `1.0.0` not release, waiting for other members comment

File: src/OrchardCore.Modules/OrchardCore.Contents/Workflows/Drivers/ContentCreatedEventDisplayDriver.cs (hunk 1 above)
Comment (agriffard) [reply src/OrchardCore.Modules/OrchardCore.Contents/Workflows/Drivers/ContentCreatedEventDisplayDriver.cs:1]: I don't know why those ones haven't been dtected as renamed like the others.

**Comment by agriffard** [comment]
Why would it be breaking changes?

**Comment by hishamco** [comment]
> Why would it be breaking changes?

Because there are public types

**Review by sfmskywalker — APPROVED** [review]

**Comment by sfmskywalker** [comment]
@hishamco Although technically speaking you are correct, in practice though I reckon that the odds of custom modules directly referencing a public display driver are really small. So seems like a non-issue to me.

**Comment by hishamco** [comment]
So if it's like that let us merge

**Pull request merged**